src/main_app/db/__init__.py

```
@@ -50,14 +50,16 @@ def create_views(_db: SQLAlchemy) -> None:
-            if table.name in existing_views:
-                continue
+            create_sql = table.info["create_query"]
+            if not table.info.get("repalce_the_view"):
+                if table.name in existing_views:
+                    continue
+
             try:
                 with conn.begin():
-                    create_sql = table.info["create_query"]
                     conn.execute(text(create_sql))
             except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)